freespace2/freespace.cpp

```diff
@@ -2047,6 +2048,9 @@ void game_get_framerate()
 	Framecount++;
 }
 
+int bmpman_count_bitmaps();
+extern int MAX_BITMAPS;
```

Comment:
This variable is already declared in the bmpman header so that header should be included instead of using an `extern` variable here.

Reply:
Oh right; I meant this to be a temporary thing just to see if it was working, and then forgot to fix it later. Let's take care of that right now...